docs/UsersGuide/source/Graphics.rst

```diff
@@ -19,58 +19,23 @@ output over the :term:`CONUS` for a number of variables, including:
 
 The Python scripts are located under ``ufs-srweather-app/ush/Python``.
 The script ``plot_allvars.py`` plots the output from a single cycle within an experiment, while 
-the script ``plot_allvars_diff.py`` plots the difference between the same cycle from two different
-experiments (e.g., the experiments may differ in some aspect such as the physics suite used). If 
+the script ``plot_allvars_diff.py`` plots the difference between the same cycle from two different experiments. If 
 plotting the difference, the two experiments must be on the same domain and available for 
-the same cycle starting date/time and forecast hours. 
+the same cycle starting date/time and forecast hours. Other parameters may differ (e.g., the experiments may use different physics suites). 
 
 Loading the Environment
 ==========================
 
-To use the plotting scripts, the appropriate environment must be loaded. The scripts require Python 3 with the ``scipy``, ``matplotlib``, ``pygrib``, ``cartopy``, and ``pillow`` packages. This Python environment has already been set up on Level 1 platforms and can be activated as follows:
-
-On Cheyenne:
-
-.. code-block:: console
-
-   module load ncarenv
-   module load conda/latest
-   conda activate /glade/p/ral/jntp/UFS_SRW_app/conda/python_graphics
-
-On Hera and Jet:
-
-.. code-block:: console
-
-   module use -a /contrib/miniconda3/modulefiles
-   module load miniconda3
-   conda activate pygraf
-
-On Orion:
```

Comment:
This graphics section will be a lot smaller in the next version since it will become part of the workflow. However, the user can still execute the plotting scripts manually if he wishes too.